R/api_url.R

```diff
@@ -92,7 +93,8 @@ api_url <- function(
       stop(
         paste(
           "You have entered an invalid endpoint, this should one of:",
-          "get-summary, get-meta, get-data or post-data"
+          "get-publication, get-data-catalogue, get-summary, get-meta,",
```

Comment:
Slight typo here, should be "get-publications" rather then "get-publication"

Reply:
Good spot! Fixed that now